en/Formulaic Privacy Notice

```diff
@@ -0,0 +1,77 @@
+**Communications.** When you contact us in connection with the Service, we may also collect personal information, such as your username, name, email address, or phone number, as well as any other personal information you may provide to us.  
+
+We also *automatically* receive certain information when you interact with the Service: 
+**Technical data.** We receive basic telemetry data by default, which we use to improve the performance, stability, and security of the Service, including information about your device, including hardware configuration and device operating system; browser information, such as browser type and settings; log files and usage data; and event information, such as errors or crashes. We may also receive additional information about your device, such as country, language, operator and OEM, which we use to prevent fraud and abuse, enforce our terms, and improve the security of the Service.   
+**Location data.** Your IP address is temporarily collected as part of our server logs, and automatically deleted after 30 days. We use it to approximate your location to comply with relevant legal and regulatory requirements, as well as for fraud and abuse prevention purposes. 
+**Interaction data.** We receive aggregate data about visits to the Formulaic website and other aggregate insights about user interactions with the Service. We use this data to better understand usage of the Service, as well as to add, improve, or remove certain features of the Service. We may also receive HTTP referral data to understand the website or domain that referred you to our site, which we only use for purposes of addressing bugs or other technical issues with our Service. We automatically delete this data after 30 days.   
+**Metrics.** We may also use cookies, device information, and IP addresses, along with clear GIFs, cookies, and third party services to securely operate the Service, help us understand how users engage with our Service, such as visits to our site and settings preferences, and provide additional functionality. We use:
+-Sign in with Google, which places a cookie on your device to assist with sign-in and authentication to the Service. When you sign into the Service, you accept the associated [Privacy Policy](https://policies.google.com/privacy?hl=en-US) and [Terms of Service](https://policies.google.com/terms) from Google.
+-Google Analytics to obtain metrics on how users engage with our websites. This helps us to improve site content. For more information about how Google uses your personal information, please visit [Google Analytics’ Privacy & Security](https://support.google.com/analytics/topic/2919631?hl=en&ref_topic=1008008&sjid=14989286036636170427-NA). You can install the [Google Analytics Opt-out Browser Add-on](https://tools.google.com/dlpage/gaoptout) to prevent data collection about your visits to the Service and prohibit data transmission to Google Analytics.
```

Comment:
Can we remove `en` in the link to automatically redirect to default language of choice?